DIRSHA256.md

```diff
@@ -0,0 +1,212 @@
+# DIRSHA256 specifications
+
+## Abstract
+
+This document describes DIRSHA256, an algorithm to serialize the content of a filesystem path using the cryptographic hash function SHA256. The cryptographic strength of DIRSHA256 depends on the properties of the SHA256 hash function.
```

Comment:
Should we already mention up here that the algorithm is parameterized?

Reply:
done.